server/controllers/paragraph.ts

```diff
@@ -0,0 +1,27 @@
+import { Request, Response } from 'express';
+import Text from '../schemas/paragraph';
+
+async function getRandomParagraph(_: Request, res: Response): Promise<void> {
+  try {
+    const randomNumber = Math.floor(Math.random() * 7191);
```

Comment:
TODO: don't use 7191, I think it's bad practice.
Would be better to get current number of quotes from database. But not sure, we should do it every time. Ask someone about best practice.